source/_integrations/mcp_server.markdown

```diff
@@ -65,58 +65,135 @@ The Model Context Protocol specification does not yet define standards
+### 2. Verify server accessibility
+
+Before proceeding, verify that your Home Assistant instance is accessible and the MCP Server integration is properly configured:
+
+```bash
+curl http://homeassistant.local:8123/mcp_server/sse
+```
+
+This should return `401: Unauthorized`, which confirms that the server is accessible and the MCP Server integration is installed.
+
+### 3. Install the MCP proxy
+
+The MCP proxy acts as a bridge between your LLM client and Home Assistant:
+
+1. Install the MCP proxy:
+   ```bash
+   uv tool install git+https://github.com/sparfenyuk/mcp-proxy
+   ```
+
+2. Verify the installation and locate the absolute path:
+   ```bash
+   which mcp-proxy
+   ```
+   Note this absolute path for later configuration steps.
+
+3. Test your configuration before setting it up in Cursor or Claude:
+   ```bash
+   env API_ACCESS_TOKEN=<YOUR TOKEN> /absolute/path/to/mcp-proxy http://homeassistant.local:8123/mcp_server/sse
+   ```
+
+   If the command runs without crashing, your setup is working correctly.
+
+### 4. Configure your client
+
+Depending on which client you're using, follow the appropriate configuration instructions:
+
+#### Claude for Desktop
+
+Claude for Desktop uses a JSON configuration file to set up MCP servers:
+
+1. Download and install [Claude for Desktop](https://claude.ai/download).
+2. Open the configuration file: Visit **Settings…** > **Developer** tab > **Edit Config**, which will open `claude_desktop_config.json`.
+3. Add a new MCP server to the JSON file:
+   ```json
+   {
+     "mcpServers": {
+       "Home Assistant": {
+         "command": "mcp-proxy",
+         "env": {
+           "API_ACCESS_TOKEN": "<YOUR KEY>"
+         },
+         "args": ["http://homeassistant.local:8123/mcp_server/sse"]
+       }
+     }
+   }
+   ```
+4. Restart Claude.
+5. You will see a connection icon {% icon "mdi:connection" %} if things are set up correctly. Clicking the icon will show enabled MCP servers including "Home Assistant".
+6. Select the prompt provided by Home Assistant and start interacting with your home.
+
+![Screenshot of Claude for Desktop adding an item to a Home Assistant To-do list](/images/integrations/mcp_server/claude-todo-list-control.png)
+
+#### Cursor Editor in the UI
+
+Cursor's UI provides a simple way to set up the MCP connection:
+
+1. From the MCP tab in your settings panel
+2. Click "Add new MCP server"
+3. Select "command" under Type
+4. Under "Command" input:
+   ```shell
+   env API_ACCESS_TOKEN=<YOUR TOKEN> /absolute/path/to/mcp-proxy http://homeassistant.local:8123/mcp_server/sse
+   ```
+
+> Cursor handles relative paths to `mcp-proxy`, while Claude requires absolute paths.
+
+#### Shared configuration using `.cursor/mcp.json`
+
+If you want to use both Claude and Cursor with the same configuration, you can create a shared configuration file:
```

Comment:
This is very interesting and useful, but feels like too much detail for the integraton documentation, and more like a community guide.

Reply:
Just to be clear, i'm referring only to this section of having a shared config.